test/floatapprox.jl

```diff
@@ -6,19 +6,19 @@
 @test 4.000000002 ≉ 4.00300002
 
 # Other tolerance levels
-@test isapprox(4.32, 4.3; rtol=0.1, atol=0.01)
-@test isapprox(1.001, 1.002; rtol=0.001, atol=0.0001)
-@test !isapprox(4.5, 4.9; rtol=0.001, atol=0.001)
+@test 4.32 ≈ 4.3 rtol=0.1 atol=0.01
+@test 1.001 ≈ 1.002 rtol=0.001 atol=0.0001
+@test 4.5 ≉ 4.9 rtol=0.001 atol=0.001
```

Comment:
I didn't want to go to wild with it since it was unclear how successful it would be. Now that people are clammoring for more widespread use, maybe we can do that 😀